sources/osg/CullVisitor.js

```diff
@@ -50,6 +50,9 @@ var CullVisitor = function () {
     this._reserveLeafStack = [ new RenderLeaf() ];
     this._reserveLeafStack.current = 0;
 
+    this._reserveCullSettingsStack = [ new CullSettings() ];
```

Comment:
Why it is different from stack matrix ? we said that we dont want to add members in array